includes/admin/class-sensei-onboarding.php

```diff
@@ -126,6 +129,91 @@ public function onboarding_page() {
 		<?php
 	}
 
+	/**
+	 * Check if current screen is selected to display the wizard notice.
+	 *
+	 * @return boolean
+	 */
+	private function is_current_screen_selected_to_wizard_notice() {
+		$screen = get_current_screen();
+
+		if ( false !== strpos( $screen->id, 'sensei-lms_page_sensei' ) ) {
+			return true;
+		}
+
+		$screens_without_sensei_prefix = [
+			'dashboard',
+			'plugins',
+			'edit-sensei_message',
+			'edit-course',
+			'edit-course-category',
+			'course_page_course-order',
+			'edit-module',
+			'course_page_module-order',
+			'edit-lesson',
+			'edit-lesson-tag',
+			'lesson_page_lesson-order',
+			'edit-question',
+			'question',
+			'edit-question-category',
+		];
+
+		return in_array( $screen->id, $screens_without_sensei_prefix, true );
+	}
+
+	/**
+	 * Onboarding wizard notice.
+	 *
+	 * @access private
+	 */
+	public function onboarding_wizard_notice() {
+		if (
+			! $this->is_current_screen_selected_to_wizard_notice()
+			|| ! get_option( self::SUGGEST_ONBOARDING_OPTION, 0 )
+			|| ! current_user_can( 'manage_sensei' )
+		) {
+			return;
+		}
+
+		$setup_url = admin_url( 'admin.php?page=' . $this->page_slug );
+
+		$skip_url = add_query_arg( 'sensei_skip_setup_wizard', '1' );
+		$skip_url = wp_nonce_url( $skip_url, 'sensei_skip_setup_wizard' );
+		?>
+		<div id="message" class="updated sensei-message sensei-connect">
+			<p><?php echo wp_kses_post( __( '<strong>Welcome to Sensei LMS</strong> &#8211; You\'re almost ready to start creating online courses!', 'sensei-lms' ) ); ?></p>
+
+			<p class="submit">
+				<a href="<?php echo esc_url( $setup_url ); ?>" class="button-primary">
+					<?php esc_html_e( 'Run the Setup Wizard', 'sensei-lms' ); ?>
+				</a>
+
+				<a class="button" href="<?php echo esc_url( $skip_url ); ?>">
+					<?php esc_html_e( 'Skip setup', 'sensei-lms' ); ?>
+				</a>
+			</p>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Skip onboarding wizard.
+	 *
+	 * @access private
+	 */
+	public function skip_setup_wizard() {
```

Comment:
See above comments re: "setup" vs. "onboarding".

Reply:
Refactoring to `setup_wizard`. So this case is correct :)